src/app/views/query/form/query-form.component.js

```diff
@@ -248,9 +299,7 @@ export default {
   bindings: {
     params: '<',
     queryMap: '<',
-    expeditions: '<', // list of expeditions
```

Comment:
you should remove the code that was injecting the expeditions. Either in `query.routes.js` or `query.component.js`